lib/url.c

```diff
@@ -2275,7 +2275,7 @@ static CURLcode parse_proxy(struct Curl_easy *data,
       goto error;
     }
     /* path will be "/", if no path was found */
-    if(strcmp("/", path)) {
+    if(strcmp("/", path) != 0) {
```

Comment:
No thanks, we compare against non-zero implicitly all through curl code on purpose. I don't think this increases readability.

Reply:
Ok. I canceled this change.